src/ui/handler/drag_rotate.js

```diff
@@ -212,14 +227,14 @@ class DragRotateHandler {
             const buttons = (e.ctrlKey ? 1 : 2),  // ? ctrl+left button : right button
                 button = (e.ctrlKey ? 0 : 2);   // ? ctrl+left button : right button
             let eventButton = e.button;
-            if (typeof InstallTrigger !== 'undefined' && e.button === 2 && e.ctrlKey &&
+            if (typeof window.InstallTrigger !== 'undefined' && e.button === 2 && e.ctrlKey &&
                 window.navigator.platform.toUpperCase().indexOf('MAC') >= 0) {
                 // Fix for https://github.com/mapbox/mapbox-gl-js/issues/3131:
                 // Firefox (detected by InstallTrigger) on Mac determines e.button = 2 when
                 // using Control + left click
                 eventButton = 0;
             }
-            return (e.type === 'mousemove' ? e.buttons & buttons === 0 : !this.isActive() && eventButton !== button);
+            return (e.type === 'mousemove' ? e.buttons && buttons === 0 : !this.isActive() && eventButton !== button);
```

Comment:
I think `(e.buttons & buttons) === 0` was indeed the intention, but in fact that prevents drag pan/rotate operations from working on Safari, which does not support the `buttons` property. Whereas `e.buttons & buttons === 0` was simply always false, so `mousemove` events were never ignored, which is harmless!

To fix this, I'm going to remove the logic for ignoring `mousemove` events. Since it was never ignoring them, and we never noticed, it must be unnecessary.